bigtable/examples/run_examples_utils.sh

```diff
@@ -45,6 +44,7 @@ function run_all_instance_admin_examples() {
   echo
   echo "Run create-instance example."
   ../examples/bigtable_samples_instance_admin create-instance "${project_id}" "${INSTANCE}" "${zone_id}" || ${ignore_unimplemented}
+  trap '../examples/bigtable_samples_instance_admin delete-instance "${project_id}" "${INSTANCE}"' EXIT
```

Comment:
The double-quoted strings are inside the outer single-quoted string; will the variables be expanded correctly?

Reply:
The stackoverflow answer and this test say yes:

https://stackoverflow.com/questions/46734691/bash-trap-with-variable-expansion

```sh
#!/usr/bin/env bash

FOO=123
echo FOO=$FOO
trap 'echo FOO="${FOO}"' EXIT

FOO=234
echo FOO=$FOO
```